apps/alert_processor/test/alert_processor/rules_engine/subscription_filter_engine_test.exs

```diff
@@ -40,8 +40,8 @@ defmodule AlertProcessor.SubscriptionFilterEngineTest do
     test "the expected subscription matches", %{alert: alert} do
       email_match = "route1,route4|weekdays|low,high|10am-2pm@test.com"
       email_no_match = "route2,route3|weekdays,sunday|low,high|10am-2pm@test.com"
-      user = insert(:user, phone_number: nil, email: email_match)
-      user2 = insert(:user, phone_number: nil, email: email_no_match)
+      user = insert(:user, communication_mode: "email", phone_number: nil, email: email_match)
+      user2 = insert(:user, communication_mode: "email", phone_number: nil, email: email_no_match)
```

Comment:
question (non-blocking): Was this test failing without this change? Or is this added so the test is more explicit? Trying to understand whether this default value of ["email" for communication_mode](https://github.com/mbta/alerts_concierge/blob/04ec1a7841f76ef578d911c3faa2d9276ab4b985/apps/alert_processor/lib/model/user.ex#L34) is as expected

Reply:
Good question. It was indeed working fine previously, adding it really was an attempt to make it more explicit. Do you find that helpful at all? If so, is it worth a comment or something clarifying that it's just intended to be explicit?

Reply:
I suppose alternately we could remove this explicit `communication_mode: "email"` bit that's redundant, but instead add a comment.

Reply:
I like that this is more explicit now! Just wasn't sure whether this was a functional change too, but good to know it is not!